README.md

```diff
@@ -13,7 +13,7 @@ The tool consumes and produces OCI-compliant container images, so you can pull a
 
 ### Requirements
 
-You need an Apple silicon Mac to run `container`. To build it, see the [BUILDING](./BUILDING.md) document.
+### **You need an Apple silicon Mac** to run `container`. To build it, see the [BUILDING](./BUILDING.md) document.
```

Comment:
Let's not use an h3 of boldface here.

Could you try putting this in an important aside (alert); see Update plugin system to work with a `libexec/container-plugins` location under the installation root.